CMakeLists.txt

```diff
@@ -163,7 +163,11 @@ CHECK_STRUCT_HAS_MEMBER("struct stat" st_mtim.tv_nsec sys/stat.h
 CHECK_STRUCT_HAS_MEMBER("struct stat" st_mtimespec.tv_nsec sys/stat.h
   HAVE_STAT_ST_MTIMESPEC_TV_NSEC LANGUAGE C)
 
-set(CEPH_MAN_DIR "share/man" CACHE STRING "Install location for man pages (relative to prefix).")
+if(FREEBSD)
+  set(CEPH_MAN_DIR "man" CACHE STRING "Install location for man pages (relative to prefix).")
```

Comment:
> Don't really know if ever anybody is going to rumage thru the CACHE looking for the STRING.

it's maintainer's job to set the right path to comply to the platform/distro's policy/convention.

> This is a rather permanent fix for building on FreeBSD

there are many settings combinations are permanent to different architecture / platform / distros, what we are offering is to make this possible and even easier to majority of our users. i'd want to avoid filling the cmake script with different combinations on all of these settings even they are permanent to their targeting platforms.